Templates/Player Character.md

```diff
@@ -53,23 +53,18 @@ Playbook XP: `clock 0/9`
 _when there are empty checkboxes below an item, it costs more than one **load**. When an item is a bullet instead of a check, it takes no load._
 
 - [ ] small weapon
-- [ ] heavy weapon
-- [ ] 
+- [ ] <input type="checkbox"/> heavy weapon
```

Comment:
Trick I found on forums to put multiple checkboxes inline. Hoping that is a little more readable than the multiline thing.

Reply:
I am not sure you're able to check these via the UI? When I tried in obsidian it just converted to HTML

Reply:
hmm, yeah. They are checkable by editing the source. I'd lean towards leaving them if you're alright with it, and maybe improving them with a small plugin of our own later.

Reply:
Thats fine for now, but yeah its a little annoying.